modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullScreenPluginTest.ts

```diff
@@ -37,7 +42,7 @@ describe('browser.tinymce.plugins.fullscreen.FullScreenPluginTest', () => {
 
   const assertApiAndLastEvent = (editor: Editor, state: boolean) => {
     assert.equal(editor.plugins.fullscreen.isFullscreen(), state, 'Editor isFullscreen state');
-    assert.equal(lastEventArgs.get().state, state, 'FullscreenStateChanged event state');
+    assert.deepEqual(firedEvents, [ 'fullscreenstatechanged', state, 'resizeeditor' ], 'Fired event state');
```

Comment:
The message here should be changed to reflect the code change maybe.
```suggestion
    assert.deepEqual(firedEvents, [ 'fullscreenstatechanged', state, 'resizeeditor' ], 'Should be expected events and state');
```